src/components/application/StatsBody.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import "./StatsBody.css";
+import fetch from "../../methods/user/stats";
+
+import "react-datepicker/dist/react-datepicker.css";
+
+export default ({switchRoute }) => {
```

Comment:
Function should be named and exported separately. See https://github.com/steveu/login-challenge/pull/3/files#r594369125

Reply:
done